examples/lighting-app/silabs/src/ZclCallbacks.cpp

```diff
@@ -56,13 +59,76 @@ void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath &
 
         // WIP Apply attribute change to Light
     }
+#ifdef SL_MATTER_RGB_LED_ENABLED
```

Comment:
We should still handle the cluter idd and print for non RGB.
You mustgate only the actual RGB stuff.
Also #idef should be avoided. Use an expected value
either SL_MATTER_RGB_LED_ENABLED is always defined to 0 or 1 and you use
`#if SL_MATTER_RGB_LED_ENABLED`
or 
`#if (defined(SL_MATTER_RGB_LED_ENABLED) && SL_MATTER_RGB_LED_ENABLED == 1)`
to reduce the chance undefined value of SL_MATTER_RGB_LED_ENABLED causing unexpected behaviour

Reply:
Added back to handle the cluster id and print for for non RGB.